compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/MultiLayerComposeScene.skiko.kt

```diff
@@ -581,7 +587,14 @@ private class MultiLayerComposeSceneImpl(
             composition?.dispose()
             composition = owner.setContent(
                 parent = this@AttachedComposeSceneLayer.compositionContext,
-                { this@AttachedComposeSceneLayer.compositionLocalContext }
+                {
```

Comment:
Better to remove this lambda completely (it has the same default value), otherwise it is difficult to read without IDE support. The comment itself can be placed above `setContent`